src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -0,0 +1,393 @@
+        (this.authorization.expiryDate >= this.authorization.date) ||
+        'Expiry Date cannot be before Authorization Date'
+    ]
+  }
+
+  /** The number of file upload componenets to display (max 5). */
+  get numUploads (): number {
+    return Math.min((this.authorization.files.length + 1), 5)
+  }
+
+  mounted (): void {
+    // set or initialize authorization object
+    this.authorization = this.getContinuationAuthorization || { files: [] } as ContinuationAuthorizationIF
+  }
+
+  /**
+   * Called when FileUploadPreview tells us whether a file is valid.
+   * This is called right before the File Selected event.
+   */
+  onFileValidity (valid: boolean): void {
+    this.fileValidity = valid
+  }
+
+  /**
+   * Called when FileUploadPreview tells us about a new or cleared file.
+   * This is called right after the File Validity event.
+   */
+  async onFileSelected (index: number, file: File): Promise<void> {
+    // reset state of file uploader to ensure not in manual error mode
+    this.fileUploadCustomErrorMsg = ''
+
+    if (file) {
+      // verify that file is valid
+      if (!this.fileValidity) {
+        this.snackbarText = 'Invalid file. It will not be saved.'
+        this.snackbar = true
+        return // don't add to list
+      }
+
+      // verify that file doesn't already exist
+      if (this.authorization.files.find(f => f.file.name === file.name)) {
+        this.snackbarText = 'File already uploaded. It will not be saved again.'
+        this.snackbar = true
+        return
+      }
+
+      // try to upload to Minio
+      let psu: PresignedUrlIF
+      try {
+        psu = await this.getPresignedUrl(file.name)
+        const res = await this.uploadToUrl(psu.preSignedUrl, file, psu.key, this.getKeycloakGuid)
+        if (!res || res.status !== 200) throw new Error()
+      } catch {
+        // put file uploader into manual error mode by passing custom error message
+        this.fileUploadCustomErrorMsg = this.UPLOAD_FAILED_MESSAGE
+
+        this.snackbarText = 'File upload error. Please try again.'
+        this.snackbar = true
+        return
+      }
+
+      // add file to array
+      this.authorization.files.push({
+        file: {
+          name: file.name,
+          lastModified: file.lastModified,
+          size: file.size
+        } as File,
+        fileKey: psu.key,
+        fileName: file.name,
+        fileUrl: psu.preSignedUrl
+      })
+    } else {
+      // remove file from array
+      this.authorization.files.splice(index, 1)
+    }
+  }
+
+  @Watch('getShowErrors')
+  @Watch('authorization.date') // because Expiry Date depends on this
+  private onShowErrors (): void {
+    if (this.getShowErrors) {
+      // validate the form and the components that don't support form validation
+      this.$refs.formRef.validate()
+      this.$refs.authorizationDateRef.validateForm()
+      this.$refs.expiryDateRef.validateForm()
+    }
+  }
+
+  @Watch('authorization', { deep: true })
```

Comment:
Whenever the authorization object reference changes, or any of its properties, this method gets called.